testing/trino-tests/src/test/java/io/trino/execution/TestTry.java

```diff
@@ -27,7 +30,7 @@
     protected QueryRunner createQueryRunner()
             throws Exception
     {
-        return MemoryQueryRunner.createQueryRunner();
+        return createMemoryQueryRunner(Map.of(), List.of());
```

Comment:
nit: missed immutable here.

I do think it may make sense to use the shorter version in tests but let's leave that for a dedicated PR.

Reply:
> I do think it may make sense to use the shorter version in tests but let's leave that for a dedicated PR.

i don't plan to have a discussion around this. 
i believe debating "List" vs "ImmutableList" in test code is pure waste of time. 
I will change this PR to be compliant.